src/docfx/lib/schema/JsonSchemaTransformer.cs

```diff
@@ -284,13 +287,19 @@ private JToken TransformScalar(ErrorBuilder errors, JsonSchema schema, FilePath
 
                 case JsonSchemaContentType.Markdown:
 
+                    var mime = _documentProvider.GetMime(file);
+                    var rootSchema = _templateEngine.GetSchema(_documentProvider.GetMime(file));
```

Comment:
We should have `rootSchema` in line 55, pass it down over having a dependency on `_templateEngine` here.